packages/regenesis-surgery/package.json

```diff
@@ -21,6 +21,7 @@
     "@discoveryjs/json-ext": "^0.5.3",
     "@eth-optimism/contracts": "^0.4.10",
     "@uniswap/v3-core-optimism": "^1.0.0-rc.0",
+    "axios": "^0.22.0",
```

Comment:
Aren't these all `devDependencies` (especially since this package is private)?

Reply:
Yes, everything should be moved into devDependencies

Reply:
removed axios and using node-fetch instead based on below comment, added that to devDependencies but will fix rest of them in a different PR so this package.json doesnt get fucked up